dev_symlinks.sh

```diff
@@ -1,9 +1,65 @@
-#!/bin/sh
+#!/bin/bash
 # Create symlinks into Git submodules, so the project can
 # be started with qmlscene directly from a source checkout
 
-ln -sf gpodder-core/src/gpodder .
-ln -sf podcastparser/podcastparser.py .
-ln -sf gpodder-ui-qml/main.py .
-ln -sf ../gpodder-ui-qml/common qml
-ln -sf minidb/minidb.py .
+# array of symlinks in form "$TARGET","$LINK_NAME"
+SYMLINKS=("gpodder-core/src/gpodder","gpodder" "podcastparser/podcastparser.py","podcastparser.py" "gpodder-ui-qml/main.py","main.py" "../gpodder-ui-qml/common","qml/common" "minidb/minidb.py","minidb.py")
```

Comment:
I think this array would be easier to maintain if each entry would be in a new line.

Reply:
I have to figure out whether that's possible, because bash arrays are a bit weird. Or I just rewrite the whole script in python.

Reply:
You can always escape new lines with `\` like this:

```bash
SYMLINKS=( \
        "gpodder-core/src/gpodder","gpodder" \
        "podcastparser/podcastparser.py","podcastparser.py" \
        "gpodder-ui-qml/main.py","main.py" \
        "../gpodder-ui-qml/common","qml/common" \
        "minidb/minidb.py","minidb.py" \
)
```